<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modify_product</class>
 <widget class="QDialog" name="modify_product">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sửa/Xóa sản phẩm</string>
  </property>
  <widget class="QTableView" name="tv_list_san_pham">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>401</width>
     <height>231</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>25</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>261</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lb_msp">
        <property name="text">
         <string>Mã sản phẩm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_tensp">
        <property name="text">
         <string>Tên sản phẩm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_phan_loai">
        <property name="text">
         <string>Phân loại</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_gia_nhap">
        <property name="text">
         <string>Giá nhập</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_gia_xuat">
        <property name="text">
         <string>Giá xuất</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_nha_cung_cap">
        <property name="text">
         <string>Nhà cung cấp</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="le_msp">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_tensp"/>
      </item>
      <item>
       <widget class="QComboBox" name="cb_phan_loai">
        <property name="toolTipDuration">
         <number>0</number>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="maxVisibleItems">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_gia_nhap">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_gia_xuat">
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_nha_cung_cap">
        <property name="toolTipDuration">
         <number>0</number>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lb_title">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sửa/Xóa sản phẩm</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>261</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pb_sua">
      <property name="text">
       <string>Sửa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_xoa">
      <property name="text">
       <string>Xóa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_huy">
      <property name="text">
       <string>Hủy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
